codegen/method.go

```diff
@@ -881,7 +881,7 @@ func (ms *MethodSpec) setTypeConverters(
 
 	respConverter.append(
 		"func convert",
-		pascalCase(ms.Name),
+		ms.DownstreamService, pascalCase(ms.Name),
```

Comment:
While conventionally service name is in pascal case, but it's not enforced. So let's apply `pascalCase` on the service name as well.

Reply:
Got it!